src/snowflake/snowpark/types.py

```diff
@@ -68,7 +68,13 @@ class DateType(_AtomicType):
 class StringType(_AtomicType):
     """String data type. This maps to the VARCHAR data type in Snowflake."""
 
-    pass
+    _MAX_LENGTH = 16777216
+
+    def __init__(self, length = 16777216) -> None:
```

Comment:
How about using `None` as the default value?

Reply:
I am using `None` as default value but it creates a difference between client and server in the meaning of uninitialized string type. On server `STRING = STRING(16777216)` but on client `StringType() != StringType(16777216)`